std/variant.d

```diff
@@ -102,6 +102,20 @@ struct This;
 
 private alias This2Variant(V, T...) = AliasSeq!(ReplaceType!(This, V, T));
 
+// We can't just use maxAlignment because no types might be specified
+// to VariantN, so handle that here and then pass along the rest.
+private template maxVariantAlignment(U...)
+if (isTypeTuple!U)
+{
+    static if (U.length == 0)
+        enum maxVariantAlignment = size_t.alignof;
+    else
+    {
+        import std.traits : maxAlignment;
```

Comment:
There's a global `import std.traits` at the top of the file.

Reply:
Removed.